app/controllers/staff/organisations_controller.rb

```diff
@@ -12,13 +12,35 @@ def show
 
     @organisation_presenter = OrganisationPresenter.new(organisation)
 
-    @project_activities = FindProjectActivities.new(organisation: organisation, user: current_user).call(eager_load_parent: false).publishable_to_iati
-    @third_party_project_activities = FindThirdPartyProjectActivities.new(organisation: organisation, user: current_user).call(eager_load_parent: false).publishable_to_iati
+    fund_ids_for_organisation_programmes = Activity.where(
+      level: :programme,
+      extending_organisation_id: organisation.id
+    ).pluck(:parent_id)
+    @funds = Activity.find(fund_ids_for_organisation_programmes)
+
+    @project_activities = FindProjectActivities.new(
+      organisation: organisation,
+      user: current_user
+    ).call(eager_load_parent: false).publishable_to_iati
+
+    @third_party_project_activities = FindThirdPartyProjectActivities.new(
+      organisation: organisation,
+      user: current_user
+    ).call(eager_load_parent: false).publishable_to_iati
```

Comment:
These variables are well-named, and it's probably worth pulling these queries out into methods so the `show` method becomes a high-level description of what's being displayed, rather than the implementation details of fetching that data.